src/main/java/codeu/model/data/User.java

```diff
@@ -37,6 +41,29 @@ public User(UUID id, String name, String password, Instant creation) {
     this.name = name;
     this.password = password;
     this.creation = creation;
+    this.messages = new LinkedList<Message>();
+    aboutUser = "";
+  }
+  
+  // adds about me information for the user 
+  public void setAbout(String about) {
+	  this.aboutUser = about;
+  }
+  
+  // returns the current about me information
+  public String getAbout() {
+	  return aboutUser;
+  }
+  
+  public void addMessage(Message newMessage) {
+	  if (messages.size() == NUMBER_OF_MESSAGES) {
```

Comment:
Maybe this isn't something to worry about, but if the NUMBER_OF_MESSAGES (the constant) to be stored in the User/displayed on the Profile change were to decrease, the number stored/displayed in the user wouldn't decrease with it.

Reply:
You are right, I removed this limitation in User and modified profile.jsp instead. Thank you!